### DISCUSSION TITLE
Share your Progress and Doubts here

### DISCUSSION THREAD

**Discussion by anandjaiswar02** [anandjaiswar02/Hactoberfest-Windows11#6] [General]
Hello community, you all can share your doubts and progress in contributing to this project here in this discussion. 
I would be happy to help you all.

**Comment by MAVERICK-VF142** [comment]
how to stay on full screen of the start menu without the animation

**Reply by anandjaiswar02** [reply]
> how to stay on full screen of the start menu without the animation

![image](https://user-images.githubusercontent.com/85798085/194752389-d41a9788-b7d5-4ee1-9bb7-ff16952976de.png)

Means just you have to do this Without clicking on the start menu button Right???

**Reply by MAVERICK-VF142** [reply]
Yes

**Reply by anandjaiswar02** [reply]
![image](https://user-images.githubusercontent.com/85798085/194752624-6def2c80-63c4-42a7-b66f-110bdab5509c.png)

Just comment these two lines

**Reply by anandjaiswar02** [reply]
Hope this helps!!
Keep coding!!

**Comment by MAVERICK-VF142** [comment]
hey can you explain me the reason and help me fixing it

**Reply by anandjaiswar02** [reply]
Hello @MAVERICK-VF142 ,
Hope you are doing well
Bug1:
![image](https://user-images.githubusercontent.com/85798085/194763727-3995cf09-3990-45ff-a505-ef72ee59eb40.png)


See for this the CSS isn't proper.


Also  for All Apps Button, we cannot put a second Page with almost same content of HTML,CSS, JavaScript.
Instead we can Make use of JavaScript on the Home Page for make it working!!

Just like you can see StartMenu, SearchMenu, Widgets, etc

**Reply by anandjaiswar02** [reply]
When User clicks on All Apps button then we can simply show just the All Apps section to the user with **JS** and thus not repeating our self.
This way we can also add cool transitions to our page and make a look of real Operating system.

**Reply by MAVERICK-VF142** [reply]
thanks will try to fix it

**Reply by anandjaiswar02** [reply]
> thanks will try to fix it

Keep coding @MAVERICK-VF142 !!
Never loose hope!
**Remember a programmer is excellent Problem Solver**. You can stackoverflow your problems to make this work!!

**Comment by anandjaiswar02** [comment]
###         > how to switch on GitHub Pages?

Go to Your forked repository settings. 
Go to Pages
Then **Branch** Select **main**

_Originally posted by @anand-3399 in https://github.com/anand-3399/Hactoberfest-Windows11/issues/9#issuecomment-1272555353_


**Remember to share the LINK of your GITHUB PAGES**
